src/app/data/constants.js

```diff
@@ -6,27 +6,24 @@ const breakpoints = {
 const itemFilters = [
   {
     type: 'location',
-    options: items => items.map(item => ({
+    options: item => ({
```

Comment:
Is there a better name for this function now? `retrieveOption`?

Reply:
Changed